geodiff/src/geodiff.cpp

```diff
@@ -688,7 +688,11 @@ int GEODIFF_makeCopySqlite( const char *src, const char *dst )
   // gets overwritten, regardless of its original content (making the API easier to use for the caller).
   if ( fileexists( dst ) )
   {
-    fileremove( dst );
+    Logger::instance().warn( "MakeCopySqlite: Removing existing destination database: " + std::string( dst ) );
```

Comment:
suggestion: 

cannot we move it to `else` branch so in the logs you have either "failed to remove ..." or "removed ..." sentence?